spacy/tests/regression/test_issue4529.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from spacy.compat import is_python2
+from spacy.gold import GoldParse
+
+
+@pytest.mark.parametrize(
+    "text,words", [("A'B C", ["A", "'", "B", "C"]), ("A-B", ["A-B"])]
+)
+@pytest.mark.skipif(is_python2, reason="I don't want to spend time on python2 support")
```

Comment:
What's the problem on Python 2? I think you just need to add unicode declarations at the top of the file? The code changes in this PR shouldn't make a difference, since it's all Cython anyways and gets compiled in a cross-compatible way.

Reply:
Thanks!